docs/components.md

```diff
@@ -43,6 +43,28 @@ See also the more [detailed documentation for list components](lists.md).
 
 Shows a pretty checkbox. The same props are accepted as for `<input type="checked" />` elements. Use `value` for whether the checkbox is checked or not, rather than `checked`. If no `id` is passed, one will be generated and used.
 
+## ColumnWrapper
+
+This is a default column wrapper available to be used by the applications.  This will ensure all applications experience the same behavior for columns used in lists.
```

Comment:
This is not for list columns, but for column layouts. Best to remove that ambiguity.